tools/hdf_converter/hdf_converter.py

```diff
@@ -9,55 +9,49 @@
 
 warnings.simplefilter('ignore')
 
-parser = optparse.OptionParser()
-parser.add_option("--dataframe", help="Name of hdf dataframe")
-parser.add_option("--table", help="Name of a table in the dataframe")
-(options, args) = parser.parse_args()
+parser = argparse.ArgumentParser()
+parser.add_argument("--dataframe", help="Name of hdf dataframe")
+parser.add_argument("--table", help="Name of a table in the dataframe")
+parser.add_argument('output')
+args = parser.parse_args()
```

Comment:
I think this part maybe has to move into main in order to be evaluated properly. This maybe causes Galaxy to fail running the tool?

Reply:
As we discussed earlier I don't think these changes are necessary since the tool works as it is. I think we can leave it in this state for now.